plat/xen/arm/entry64.S

```diff
@@ -317,12 +317,11 @@ ENTRY(_libxenplat_setup_cpu)
 	 *   NORMAL_WB               11111111 (0xff)
 	 *   NORMAL_WT               10111011 (0xbb)
 	 */
-	ldr     x0, =(MAIR_DEVICE_nGnRnE | \
-	             MAIR_DEVICE_nGnRE | \
-	             MAIR_DEVICE_GRE | \
-	             MAIR_NORMAL_NC | \
-	             MAIR_NORMAL_WB | \
-	             MAIR_NORMAL_WT)
+	ldr     x0, =(MAIR(MAIR_DEVICE_nGnRnE, MT_DEVICE_nGnRnE) | \
+	             MAIR(MAIR_DEVICE_nGnRE, MT_DEVICE_nGnRE) | \
```

Comment:
I think the last sentence can be re-phrased to be more clear:
```
This commit moves MAIR_* mask left on MT_* * 8 bits so that MAIR masks won't overlap.
```